kasa/smartdevice.py

```diff
@@ -329,20 +329,36 @@ async def _modular_update(self, req: dict) -> None:
             )
             self.add_module("emeter", Emeter(self, self.emeter_type))
 
+        request_list = []
+        est_response_size = 1024 if "system" in req else 0
+        modules_to_skip = MODEL_MODULE_SKIPLIST.get(self.model, [])
```

Comment:
Given that Brian reported it is working fine with this PR and w/o the skiplist, let's remove the skiplist altogether in this PR!